File: Exercise/Hard/12. Centrifuge Balance Checker/Code/Centrifuge Balance Checker.js
function cFuge(n, k) {
    if (n == 1 || k == 1) {
        return false;
    }
    if (n % 2 != 0) {
        return n === k;
    }
    return (n - k) % 2 === 0;
}

function main() {
    const input = "5,3";
    const parts = input.split(',');

    if (parts.length === 2) {
        const n = parseInt(parts[0], 10);
        const k = parseInt(parts[1], 10);

        if (cFuge(n, k)) {
            console.log("true");
        } else {
            console.log("false");
        }
    }
}

main();

File: Exercise/Hard/12. Centrifuge Balance Checker/Code/Centrifuge Balance Checker.java
import java.util.Scanner;

public class Main {
    public static boolean cFuge(int n, int k) {
        if (n == 1 || k == 1) {
            return false;
        }
        if (n % 2 != 0) {
            return n == k;
        }
        return (n - k) % 2 == 0;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();

        String[] parts = input.split(",");
        if (parts.length == 2) {
            int n = Integer.parseInt(parts[0]);
            int k = Integer.parseInt(parts[1]);

            if (cFuge(n, k)) {
                System.out.println("true");
            } else {
                System.out.println("false");
            }
        }
        scanner.close();
    }
}



